jcache/embedded/src/test/resources/infinispan_uri.xml

```diff
@@ -0,0 +1,11 @@
+<?xml version="1.0" encoding="UTF-8"?>
+<infinispan xmlns="urn:infinispan:config:7.2">
+
+    <cache-container default-cache="default">
+        <local-cache name="foo" statistics="true">
+            <eviction max-entries="200"/>
```

Comment:
You might remove `eviction` and `expiration` - they have nothing in common with initializing procedure.